runtime/src/lib.rs

```diff
@@ -1975,12 +1975,14 @@ impl kensetsu::Config for Runtime {
     type WeightInfo = kensetsu::weights::SubstrateWeight<Runtime>;
 }
 
+#[cfg(feature = "wip")] // Apollo
```

Comment:
`wip` is only for dev env.
Test env & Stage env works with `ready-to-test` flag.
Do you want to enable Apollo for dev only or for Stage as well?

Reply:
Thank you for noticing